src/v2/recommender.py

```diff
@@ -123,9 +125,10 @@ def filter_versions(epv_list, input_stack, external_request_id=None, rec_type=No
 
                 if latest_version and latest_version == version:
                     pkg_dict, new_dict, filtered_comp_list = GraphDB.add_version_to_filtered_list(
-                        epv=epv, key='latest_version', val=latest_version, pkg_dict=pkg_dict,
-                        new_dict=new_dict, filtered_comp_list=filtered_comp_list,
-                        semversion_tuple=semversion_tuple, input_stack_tuple=input_stack_tuple)
+                        eri=external_request_id, epv=epv, key='latest_version',
```

Comment:
Why can't just measure the elapsed time from the caller of `GraphDB.add_version_to_filtered_list` instead of doing it inside?